cmd/ctl/pkg/status/certificate/certificate.go

```diff
@@ -0,0 +1,170 @@
+		Example: example,
+		Run: func(cmd *cobra.Command, args []string) {
+			cmdutil.CheckErr(o.Validate(args))
+			cmdutil.CheckErr(o.Complete(factory))
+			cmdutil.CheckErr(o.Run(args))
+		},
+	}
+	return cmd
+}
+
+// Validate validates the provided options
+func (o *Options) Validate(args []string) error {
+	if len(args) < 1 {
+		return errors.New("the name of the Certificate has to be provided as argument")
+	}
+	if len(args) > 1 {
+		return errors.New("only one argument can be passed in: the name of the Certificate")
+	}
+	return nil
+}
+
+// Complete takes the factory and infers any remaining options.
+func (o *Options) Complete(f cmdutil.Factory) error {
+	var err error
+
+	o.Namespace, _, err = f.ToRawKubeConfigLoader().Namespace()
+	if err != nil {
+		return err
+	}
+
+	o.RESTConfig, err = f.ToRESTConfig()
+	if err != nil {
+		return err
+	}
+
+	o.CMClient, err = cmclient.NewForConfig(o.RESTConfig)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Run executes status certificate command
+func (o *Options) Run(args []string) error {
+	ctx := context.TODO()
+	crtName := args[0]
+
+	crt, err := o.CMClient.CertmanagerV1alpha2().Certificates(o.Namespace).Get(ctx, crtName, metav1.GetOptions{})
+	if err != nil {
+		return fmt.Errorf("error when getting Certificate resource: %v", err)
+	}
+
+	fmt.Fprintf(o.Out, fmt.Sprintf("Name: %s\nNamespace: %s\n", crt.Name, crt.Namespace))
+
+	// Get necessary info from Certificate
+	// Output one line about each type of Condition that is set.
+	conditionMsg := ""
+	for _, con := range crt.Status.Conditions {
+		conditionMsg += fmt.Sprintf("  %s: %s, Reason: %s, Message: %s\n", con.Type, con.Status, con.Reason, con.Message)
+	}
+	if conditionMsg == "" {
+		conditionMsg = "  No Conditions set\n"
+	}
+	fmt.Fprintf(o.Out, fmt.Sprintf("Conditions:\n%s", conditionMsg))
+
+	// TODO: Get CR from certificate if exists. I think I can just look for it without caring what condition is
+	// What about timing issues? When I query condition it's not ready yet, but then looking for crn it's finished and deleted
+
+	dnsNames := formatStringSlice(crt.Spec.DNSNames)
+	fmt.Fprintf(o.Out, fmt.Sprintf("DNS Names:\n%s", dnsNames))
+
+	issuerFormat := `Issuer:
+  Name: %s
+  Kind: %s`
+	fmt.Fprintf(o.Out, fmt.Sprintf(issuerFormat+"\n", crt.Spec.IssuerRef.Name, crt.Spec.IssuerRef.Kind))
+
+	fmt.Fprintf(o.Out, fmt.Sprintf("Secret Name: %s\n", crt.Spec.SecretName))
+
+	fmt.Fprintf(o.Out, fmt.Sprintf("Not Before: %s\n", formatTimeString(crt.Status.NotBefore)))
+	fmt.Fprintf(o.Out, fmt.Sprintf("Not After: %s\n", formatTimeString(crt.Status.NotAfter)))
+	fmt.Fprintf(o.Out, fmt.Sprintf("Renewal Time: %s\n", formatTimeString(crt.Status.RenewalTime)))
+
+	// TODO: print information about secret
+	return nil
+}
+
+func formatStringSlice(strings []string) string {
+	result := ""
+	for _, string := range strings {
+		result += "- " + string + "\n"
+	}
+	return result
+}
+
+func formatTimeString(t *metav1.Time) string {
+	if t != nil {
```

Comment:
nit: i am more a fan in Go to have the if statements define the non success path so when reading the code your mind can go skip over these to understand what a function does quickly. 
```go 
func formatTimeString(t *metav1.Time) string {
	if t == nil {
            return "<none>"
	}
        return t.Time.Format(time.RFC3339)
}